uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -411,7 +411,8 @@
       </xsl:if>
 
       <!-- Remove Icon -->
-      <xsl:if test="@unremovable='false' and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
+      <!-- note: deleteAllowed will either be false or not present if set from the admin ui -->
+      <xsl:if test="(@unremovable='false' and not(@dlm:deleteAllowed='false')) and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
```

Comment:
I've got some additional work that's not quite ready to commit that basically deprecates the "unremovable" flag.  Unfortunately, it's trickier than I expected it to be so I couldn't quite get it finished in the time I had allotted.   I filed a new bug at:  UP-4399 and if I can find the time I hope to finish that up. I had planned on removing this then.   For now, just concerned about this breaking existing configurations (eg. reimporting existing portlet-definition files).